src/widgets/alumni/ui/alumni.tsx

```diff
@@ -1,41 +1,26 @@
+import classNames from 'classnames/bind';
 import { alumni } from '../constants';
-import { useWindowSize } from '@/shared/hooks/use-window-size';
 import { Image } from '@/shared/ui/image';
 import { Paragraph } from '@/shared/ui/paragraph';
 import { WidgetTitle } from '@/shared/ui/widget-title';
 
-import './alumni.scss';
+import styles from './alumni.module.scss';
 
-export interface AlumniProps {
-  id: string;
-  image: string;
-}
+const cx = classNames.bind(styles);
 
 export const Alumni = () => {
-  const size = useWindowSize();
-
-  let alumniArr = [];
-
-  if (size.width <= 1440 && size.width > 810) {
-    alumniArr = alumni.slice(0, 12);
-  } else if (size.width <= 810) {
-    alumniArr = alumni.slice(0, 6);
-  } else {
-    alumniArr = [...alumni];
-  }
-
   return (
-    <article className="alumni container">
-      <section className="alumni content">
+    <article className={cx('container')}>
+      <section className={cx('content', 'info-wrapper')}>
```

Comment:
```suggestion
      <section className={cx('content', 'alumni')}>
```

Reply:
I suggest using more specific classes for sections, class `info-wrapper` is used in several widgets, using more specific names will make it easier to navigate the code base